tutorials/v7/df104.py

```diff
@@ -104,11 +104,16 @@
 upper_pad = c.AddPad(RPadPos(0,0), RPadExtent(1, 0.65))
 
 upper_frame = upper_pad.AddFrame()
-upper_frame.AttrMargins().SetBottom(0).SetLeft(0.14).SetRight(0.05)
-upper_frame.AttrX().SetHideLabels()
+upper_frame.margins.bottom = 0
+upper_frame.margins.left = 0.14
+upper_frame.margins.right = 0.05
+upper_frame.x.labels.hide = True
```

Comment:
Those examples show where this pattern hurts. Many more examples are below. Can we use
```suggestion
upper_frame.margins = (.bottom = 0, .left = 0.14, .right = 0.05)
upper_frame.x.labels.hide = True
```
At least in C++? :-)